test/e2e/standard/sanitychecker.go

```diff
@@ -37,9 +38,12 @@ type DeepTestInterface interface {
 }
 
 type SanityChecker struct {
-	Log    *logrus.Entry
-	cs     *internalapi.OpenShiftManagedCluster
-	Client *openshift.ClientSet
+	Log          *logrus.Entry
+	cs           *internalapi.OpenShiftManagedCluster
+	Client       *openshift.ClientSet
+	FakeRPClient *azure.Client
+	RealRPClient *azure.Client
+	AnyRPClient  *azure.Client
```

Comment:
Boo, it seems like we don't have a single e2e test which can run against the realrp and against the fakerp :-(
Remove AnyRPClient for now.